pulsar-common/src/main/java/org/apache/pulsar/common/naming/TopicName.java

```diff
@@ -331,11 +331,21 @@ public String getPersistenceNamingEncoding() {
      * @return
      */
     public String getLookupName() {
+        return getLookupName(null);
+    }
+
+    public String getLookupName(String listenerName) {
```

Comment:
It's better to build the url in the pulsar-admin, it's not suitable to use the TopicName to generate the request URL.